src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+    private String color;
+
+    public IsoscelesTrapezoid(Color color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * (base1 + base2) * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: IsoscelesTrapezoid, area: " + getArea()
+                + " sq. units, base1: " + base1
+                + " units, base2: " + base2
+                + " units, height: " + height
+                + " units, color: " + color);
```

Comment:
Use name() for getting String representation of enum constants. Replace the concatenation of the 'color' field with the 'getColor().name()' method call.